relay-server/src/utils/native.rs

```diff
@@ -192,6 +192,23 @@ pub fn process_minidump(event: &mut Event, data: &[u8]) {
         }
     };
 
+    let client_sdk_name = if minidump.get_stream::<MinidumpCrashpadInfo>().is_ok() {
+        "minidump.crashpad"
+    } else if minidump
+        .get_stream::<minidump::MinidumpBreakpadInfo>()
+        .is_ok()
+    {
+        "minidump.breakpad"
+    } else {
+        "minidump.unknown"
+    };
+
+    // Add sdk information for analytics.
+    event.client_sdk.get_or_insert_with(|| ClientSdkInfo {
```

Comment:
will this override the SDK set in the client? Or this only affects minidumps from the minidump endpoint?

Meaning if it's `sentry-native` attaching a minidump from crashpad, will it no longer send `sentry-native`?